File: 프로그래머스/lv0/120824. 짝수 홀수 개수/짝수 홀수 개수.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <string.h>

int* solution(int num_list[], size_t num_list_len) {
    int* answer = (int*)calloc(2, sizeof(int));
    if (answer == NULL) return NULL;
    
    for (int i = 0; i < num_list_len; i++) {
        if (num_list[i] % 2 == 0) answer[0]++;
        else answer[1]++;
    }
    
    return answer;
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        
        int len = num_list.length;
        int a = 0;
        for (int num : num_list) {
            if (num % 2 == 0) {
                a++;
            }
        }
        
        answer[0] = a;
        answer[1] = len - a;
        
        return answer;
    }
}